src/nets/e2e_asr_attctc_th.py

```diff
@@ -26,9 +26,13 @@
 from e2e_asr_common import end_detect
 from e2e_asr_common import label_smoothing_dist
 
+import matplotlib
+matplotlib.use('Agg')
+
 CTC_LOSS_THRESHOLD = 10000
 CTC_SCORING_RATIO = 1.5
 MAX_DECODER_OUTPUT = 5
+MAX_SHOW_ATTENTION = 1
```

Comment:
how about controlling this as an option? For example, `--save-attention` can accept the number of attention images (default 0).